tests/sandbox/sandbox.sh

```diff
@@ -10,6 +10,21 @@ PATH_HERE=$(dirname "$(realpath -s "$0")")
 VERSION_FILE="${PATH_HERE}"/../share/VERSION
 GOV_SCRIPT="${PATH_HERE}"/sandbox_gov.lua
 
+is_package_specified=false
+for item in "$@" ; do
+    if [ "$item" == "-p" ] || [ "$item" == "--package" ]; then
+        is_package_specified=true
+    fi
+done
+
+extra_args=("$@")
+if [ ${is_package_specified} == false ] && [ -f "${VERSION_FILE}" ]; then
```

Comment:
This change makes behaviour different for build vs install tree when only `--js-app-bundle` is given because the extra args here are added *after* the user args, overriding them and starting the wrong app.